selendroid-server/src/main/java/io/selendroid/server/model/DefaultAppiumUiAutomatorDriver.java

```diff
@@ -0,0 +1,288 @@
+            copy.put(PLATFORM, "android");
+            copy.put(SUPPORTS_ALERTS, true);
+            copy.put(SUPPORTS_JAVASCRIPT, true);
+            copy.put(SUPPORTS_NETWORK_CONNECTION, true);
+            copy.put("version", serverInstrumentation.getServerVersion());
+            copy.put(ACCEPT_SSL_CERTS, true);
+            SelendroidLogger.info("capabilities: " + copy);
+            return copy;
+        } catch (JSONException e) {
+            throw new SelendroidException(e);
+        }
+    }
+
+    @Override
+    public JSONObject getFullWindowTree() {
+        return null;
+    }
+
+    @Override
+    public String getWindowSource() {
+        return null;
+    }
+
+    @Override
+    public String initializeSession(JSONObject desiredCapabilities) {
+        //Temporary solution for session creation and this needs to be modified.
+        SecureRandom random = new SecureRandom();
+        return new BigInteger(130, random).toString(32);
+    }
+
+    @Override
+    public void stopSession() {
+
+    }
+
+    @Override
+    public void switchContext(String type) {
+
+    }
+
+    @Override
+    public byte[] takeScreenshot() {
+        return new byte[0];
+    }
+
+    @Override
+    public Keyboard getKeyboard() {
+        return null;
+    }
+
+    @Override
+    public String getTitle() {
+        return null;
+    }
+
+    @Override
+    public void get(String url) {
+
+    }
+
+    @Override
+    public TouchScreen getTouch() {
+        return null;
+    }
+
+    @Override
+    public void addCookie(String url, Cookie cookie) {
+
+    }
+
+    @Override
+    public void deleteCookie(String url) {
+
+    }
+
+    @Override
+    public void deleteNamedCookie(String url, String name) {
+
+    }
+
+    @Override
+    public Set<Cookie> getCookies(String url) {
+        return null;
+    }
+
+    @Override
+    public Object executeScript(String script, JSONArray args) {
+        return null;
+    }
+
+    @Override
+    public Object executeScript(String script, Object... args) {
+        return null;
+    }
+
+    @Override
+    public Object executeAsyncScript(String script, JSONArray args) {
+        return null;
+    }
+
+    @Override
+    public String getContext() {
+        return null;
+    }
+
+    @Override
+    public Set<String> getContexts() {
+        return null;
+    }
+
+    @Override
+    public Dimension getWindowSize() {
+        return null;
+    }
+
+    @Override
+    public void setFrameContext(Object index) throws JSONException {
+
+    }
+
+    @Override
+    public void back() {
+
+    }
+
+    @Override
+    public void forward() {
+
+    }
+
+    @Override
+    public void refresh() {
+
+    }
+
+    @Override
+    public boolean isAlertPresent() {
+        return false;
+    }
+
+    @Override
+    public String getAlertText() {
+        return null;
+    }
+
+    @Override
+    public void setAlertText(CharSequence... keysToSend) {
+
+
+    }
+
+    @Override
+    public void acceptAlert() {
+
```

Comment:
I think these should throw a not implemented exception so we know when they're called.